src/tests/unittests/validation/DepthStencilStateValidationTest.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright 2017 The NXT Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ValidationTest.h"
+
+class DepthStencilStateValidationTest : public ValidationTest {
+};
+
+TEST_F(DepthStencilStateValidationTest, Creation) {
+    // Success
+    nxt::DepthStencilState buf0 = AssertWillBeSuccess(device.CreateDepthStencilStateBuilder())
```

Comment:
nit: you could even do things like this so you can avoid naming each object differently.

```
{
    nxt::DepthStencilState ds = AssertWillBeSuccess(blah blah)
        .blah
        .foo
        .GetResult();
}
```